<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardForm</class>
 <widget class="QWidget" name="KeyboardForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>22</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>3</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="9" colspan="3">
          <widget class="QPushButton" name="btn_4">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>4 $</string>
           </property>
          </widget>
         </item>
         <item row="3" column="9" colspan="3">
          <widget class="QPushButton" name="btn_X">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>c</string>
           </property>
          </widget>
         </item>
         <item row="1" column="7" colspan="3">
          <widget class="QPushButton" name="btn_E">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>e</string>
           </property>
          </widget>
         </item>
         <item row="0" column="33" colspan="3">
          <widget class="QPushButton" name="btn_Add">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>= +</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5" colspan="3">
          <widget class="QPushButton" name="btn_S">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>s</string>
           </property>
          </widget>
         </item>
         <item row="0" column="36" colspan="2">
          <widget class="QPushButton" name="btn_Delete">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="35" colspan="3">
          <widget class="QPushButton" name="btn_Enter">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="6" colspan="3">
          <widget class="QPushButton" name="btn_C">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="1" column="31" colspan="3">
          <widget class="QPushButton" name="btn_Kuo_L">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>[ {</string>
           </property>
          </widget>
         </item>
         <item row="2" column="11" colspan="3">
          <widget class="QPushButton" name="btn_F">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>f</string>
           </property>
          </widget>
         </item>
         <item row="0" column="24" colspan="3">
          <widget class="QPushButton" name="btn_9">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>9 (</string>
           </property>
          </widget>
         </item>
         <item row="2" column="23" colspan="3">
          <widget class="QPushButton" name="btn_K">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>k</string>
           </property>
          </widget>
         </item>
         <item row="0" column="21" colspan="3">
          <widget class="QPushButton" name="btn_8">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>8 *</string>
           </property>
          </widget>
         </item>
         <item row="0" column="27" colspan="3">
          <widget class="QPushButton" name="btn_0">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>0 )</string>
           </property>
          </widget>
         </item>
         <item row="0" column="12" colspan="3">
          <widget class="QPushButton" name="btn_5">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>5 %</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" colspan="3">
          <widget class="QPushButton" name="btn_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>2 @</string>
           </property>
          </widget>
         </item>
         <item row="2" column="35" colspan="3">
          <widget class="QPushButton" name="btn_Shift">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>shift</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="20" colspan="3">
          <widget class="QPushButton" name="btn_J">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>j</string>
           </property>
          </widget>
         </item>
         <item row="2" column="29" colspan="3">
          <widget class="QPushButton" name="btn_Semicolon">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>; :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QPushButton" name="btn_Q">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>q</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="18" colspan="3">
          <widget class="QPushButton" name="btn_N">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>n</string>
           </property>
          </widget>
         </item>
         <item row="2" column="32" colspan="3">
          <widget class="QPushButton" name="btn_Yinhao">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>' &quot;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="30" colspan="3">
          <widget class="QPushButton" name="btn_Minus">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>- _</string>
           </property>
          </widget>
         </item>
         <item row="3" column="15" colspan="3">
          <widget class="QPushButton" name="btn_B">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>b</string>
           </property>
          </widget>
         </item>
         <item row="3" column="30" colspan="3">
          <widget class="QPushButton" name="btn_Question">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>/ ?</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3" colspan="3">
          <widget class="QPushButton" name="btn_Z">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>z</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="3">
          <widget class="QPushButton" name="btn_A">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>a</string>
           </property>
          </widget>
         </item>
         <item row="2" column="8" colspan="3">
          <widget class="QPushButton" name="btn_D">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>d</string>
           </property>
          </widget>
         </item>
         <item row="1" column="19" colspan="3">
          <widget class="QPushButton" name="btn_U">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>u</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="btn_Capslock">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="34" colspan="3">
          <widget class="QPushButton" name="btn_Kuo_R">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>] }</string>
           </property>
          </widget>
         </item>
         <item row="0" column="18" colspan="3">
          <widget class="QPushButton" name="btn_7">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>7 &amp;&amp;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="13" colspan="3">
          <widget class="QPushButton" name="btn_T">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>t</string>
           </property>
          </widget>
         </item>
         <item row="2" column="17" colspan="3">
          <widget class="QPushButton" name="btn_H">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>h</string>
           </property>
          </widget>
         </item>
         <item row="3" column="12" colspan="3">
          <widget class="QPushButton" name="btn_V">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>v</string>
           </property>
          </widget>
         </item>
         <item row="3" column="24" colspan="3">
          <widget class="QPushButton" name="btn_Comma">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>, &lt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="14" colspan="3">
          <widget class="QPushButton" name="btn_G">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>g</string>
           </property>
          </widget>
         </item>
         <item row="3" column="27" colspan="3">
          <widget class="QPushButton" name="btn_Period">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>. &gt;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="33" colspan="2">
          <widget class="QPushButton" name="btn_Space">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="28" colspan="3">
          <widget class="QPushButton" name="btn_P">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>p</string>
           </property>
          </widget>
         </item>
         <item row="0" column="15" colspan="3">
          <widget class="QPushButton" name="btn_6">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>6 ^</string>
           </property>
          </widget>
         </item>
         <item row="1" column="10" colspan="3">
          <widget class="QPushButton" name="btn_R">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>r</string>
           </property>
          </widget>
         </item>
         <item row="2" column="26" colspan="3">
          <widget class="QPushButton" name="btn_L">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>l</string>
           </property>
          </widget>
         </item>
         <item row="1" column="22" colspan="3">
          <widget class="QPushButton" name="btn_I">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>i</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6" colspan="3">
          <widget class="QPushButton" name="btn_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>3 #</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QPushButton" name="btn_1">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>1 !</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4" colspan="3">
          <widget class="QPushButton" name="btn_W">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>w</string>
           </property>
          </widget>
         </item>
         <item row="3" column="21" colspan="3">
          <widget class="QPushButton" name="btn_M">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
         <item row="1" column="16" colspan="3">
          <widget class="QPushButton" name="btn_Y">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QPushButton" name="btn_En">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>中/英</string>
           </property>
          </widget>
         </item>
         <item row="1" column="25" colspan="3">
          <widget class="QPushButton" name="btn_O">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>o</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>533</width>
       <height>131</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
